.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,77 @@
+name: Scan pmm server and client docker images
+
+on:
+  workflow_dispatch:
+    inputs:
+      server_image:
+        description: "pmm-server docker image"
+        required: true
+        type: string
+      client_image:
+        description: "pmm-client docker image"
+        required: true
+        type: string
+      sha:
+        description: "SHA (leave empty if running manually, default - 'null')"
+        required: false
+        type: string
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image: ${{ inputs.server_image }}
+          - image: ${{ inputs.client_image }}
+
+    steps:
+      - name: Create status check
+        if: ${{ env.SHA != 'null' }}
```

Comment:
It seems to be awkward to use `null` for a string variable? Why not check for an empty string?

Reply:
Yeah, there should be a check for empty string instead of 'null'. @yurkovychv ?